src/converse-chatview.js

```diff
@@ -1072,6 +1082,16 @@ converse.plugins.add('converse-chatview', {
                 return this;
             },
 
+            /**
+             * Remove chat state notifications for a given sender JID.
+             * @private
+             * @method _converse.ChatBoxView#clearChatStateForSender
+             * @param {string} sender - The sender of the chat state
+             */
+            clearChatStateForSender (sender) {
+                sizzle(`.chat-state-notification[data-csn="${sender}"]`, this.content).forEach(u.removeElement);
+            },
+
```

Comment:
Can you please also call `clearChatStateForSender` in `showMessage`?

Reply:
Already handled in this commit 😁

Unless there's another `showMessage` I need to search for? I see another one in the omemo view, but it invokes the one here.

Reply:
Sorry, I missed the relevant line.